test/e2e/autotls/auto_tls_test.go

```diff
@@ -75,15 +74,51 @@ func TestPerKsvcCert_localCA(t *testing.T) {
 	testingress.RuntimeRequest(t, httpsClient, "https://"+objects.Service.Status.URL.Host)
 }
 
+// To run this test locally with cert-manager, you need to
+// 1. Install cert-manager from `third_party/` directory.
+// 2. Run the command below to do the configuration:
+// kubectl apply -f test/config/autotls/certmanager/selfsigned/
+func TestPerNamespaceCert_localCA(t *testing.T) {
+	clients := e2e.Setup(t)
+	disableNamespaceCertWithWhiteList(t, clients, sets.NewString(test.ServingNamespace))
+	defer disableNamespaceCertWithWhiteList(t, clients, sets.String{})
+
+	cancel := turnOnAutoTLS(t, clients)
```

Comment:
do we need CleanupOnInterrupt for this cancel as well?

Reply:
I think we need it. Done.